stable/cosbench/templates/driver-statefulset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: {{ template "cosbench-driver.fullname" . }}
+  labels:
+    app: {{ template "cosbench.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  serviceName: {{ template "cosbench-driver.fullname" . }}
+  replicas: {{ .Values.driver.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "cosbench.name" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
afaik labels are immutable in this section (try changing the version locally and do an upgrade and test) - so labels with non-chaning values between upgrades would be needed

Reply:
Yeah this would definitely cause upgrade issues because of the chart version.

Reply:
Not in the spec.template.metadata.labels, but they are immutable in the `spec.selector.matchLabels`, which has now been added to satisfy `apps/v1`.